src/components/approval-type/ApprovalType.vue

```diff
@@ -20,12 +20,12 @@
         >
           <!-- COURT ORDER section -->
           <template v-if="!isCourtOrderRadio">
-            <span class="v-label ml-2">{{ getRadioText(ApprovalTypes.VIA_COURT_ORDER) }}</span>
+            <span class="v-label ml-2 mb-2">{{ getRadioText(ApprovalTypes.VIA_COURT_ORDER) }}</span>
           </template>
           <template v-else>
             <v-radio
               id="court-order-radio"
-              class="mb-0"
+              class="mb-n3"
```

Comment:
Could you provide screenshots of what both these changes do, please? You should be able to run the component in Storybook and take snaps of that... or use this component right in the app (you can temporarily change it in situ in the node_modules folder).